components/block/team.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="bg-white ">
+    <div
+      class="max-w-7xl mx-auto py-12 px-4 text-center sm:px-6 lg:px-8 lg:py-24"
+    >
+      <div class="space-y-22 sm:space-y-22">
+        <div class="space-y-5 sm:mx-auto sm:max-w-xl sm:space-y-4 lg:max-w-5xl">
+          <h2 class="text-3xl mb-10 font-extrabold tracking-tight sm:text-4xl">
+            Our Team
+          </h2>
+        </div>
+        <ul
+          role="list"
+          class="mx-auto grid grid-cols-2 gap-x-4 gap-y-8 sm:grid-cols-4 md:gap-x-6 lg:max-w-5xl lg:gap-x-8 lg:gap-y-12 xl:grid-cols-4"
+        >
+          <li v-for="person in people" :key="person.name">
+            <div class="space-y-4">
```

Comment:
Please pack each repeatable code as a component. 

Consult with @amanuela97 and check his similar code [here](https://github.com/wehelplive/wehelp.live/pull/37/files#diff-45423db113ce0550e32bac41fa7f598404d49d34195361302ca8f51781d469d1R4)

Reply:
Please take look at a new code, and specify the exact elements on daily.

Reply:
Just same as Figma components, every repeatable code should be component

Reply:
1) person card: name, photo, role. 2) buttons ?

Reply:
fixed